modules/messages/server/models/message.server.model.js

```diff
@@ -39,6 +41,12 @@ var MessageSchema = new Schema({
   }
 });
 
+// This index is useful when searching Messages by both userFrom and userTo
+// Is probably not necessary thanks to Index Intersection
+// https://docs.mongodb.com/manual/core/index-intersection/#index-intersection-and-compound-indexes
+//
+// MessageSchema.index({ userFrom: 1, userTo: -1 });
+
```

Comment:
I think this would be useful for `$and` queries but since we have only `$or` queries everywhere, let's drop these lines out. After that I'll merge.